### DISCUSSION TITLE
Why does wildcard question mark not match anything?

### DISCUSSION THREAD

**Discussion by mbirth** [mshibanami/redirect-web#9] [Questions] [answered]
When setting match mode to *Wildcard (\*, ?)*, I'd expect the `?` to match any single character. Thus a match pattern of:

    https://www.google.com/maps?*

should have the character directly after `maps` in `$1` and everything else in `$2` and thus should also match e.g. `https://www.google.com/mapsxyz`.

But it seems the `?` isn't implemented in wildcard mode?

**Comment by mshibanami** [comment, marked as answer]
Hi @mbirth, sorry for the late reply, and so sorry, this is a bug, and I fixed it on the latest version 5.0.0. Would you please try it?

Thank you so much for your report!

**Comment by khad** [comment]
Haha! That's why I had to set one of my rules to `$2` instead of the `$1` I was expecting. I forgot about `?` as a wildcard.
